pkg/expr/nodes.go

```diff
@@ -258,6 +261,24 @@ func (dn *DSNode) Execute(ctx context.Context, now time.Time, _ mathexp.Vars, s
 		return mathexp.Results{}, QueryError{RefID: dn.refID, Err: response.Error}
 	}
 
+	if !s.features.IsEnabled(featuremgmt.FlagDisableSSEDataplane) {
+		k, use, err := shouldUseDataplane(response.Frames)
+		if use {
+			if err != nil {
+				var vw *sdata.VersionWarning
+				if errors.As(err, &vw) {
+					logger.Warn("attempting to read mismatched version dataplane data", "error", err)
+				}
+			}
+			logger.Debug("handling SSE data source query through dataplane", "query", dn.refID)
```

Comment:
Please update log messages to start with capital letter to follow the guidelines https://github.com/grafana/grafana/blob/main/contribute/backend/instrumentation.md#usage

Reply:
Also, let's log the kind of data we received. Ideally, including the format.